lib/context.js

```diff
@@ -1,26 +0,0 @@
-/**
- * Context
- *
- * Temporary file that contains things that shouldn't be in this module
- */
-
-// contextual information we need to do the next stage of processing
-var resourceContext = {
```

Comment:
Now that GQL has a standard way of handling joins via `.relations()` and also has delegated actual aggregation back to knex, what was done here in context.js can be done within or around the bookshelf models themselves or in a search service which knows how it wants to interpret and handle aliases, etc. But I don't think it should be in the core of GQL.